src/main/resources/config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "CraftingGUITOGGLE": 0,
+  "CraftingListLength": 10,
+  "FlippingListLength": 10
+}
```

Comment:
I don't believe this config is going to save. Before release, I'll properly load the config file that's already being saved.

In the meantime, let's rename these using your IDE again:
- 'craftingToggle'
- 'craftingLength'
- 'suggesterLength'